src/main/kotlin/uk/gov/dwp/dataworks/model/Model.kt

```diff
@@ -3,10 +3,7 @@ package uk.gov.dwp.dataworks.model
 import com.fasterxml.jackson.annotation.JsonCreator
 
 data class Model @JsonCreator constructor(
-        val ecsClusterName: String,
-        val userName: String,
-        val emrClusterHostName: String,
-        val albName: String,
+        val userName: String?,
```

Comment:
This is nullable so that we can use the JWT token username for the /connect endpoint and not require it for the /deployusercontainers endpoint, as Dragos requested.